mesos-client/src/main/scala/com/mesosphere/mesos/client/MesosClient.scala

```diff
@@ -320,45 +327,50 @@ object MesosClient extends StrictLogging with StrictLoggingFlow {
 /**
   *
   */
-class MesosClientImpl(
-    sharedKillSwitch: SharedKillSwitch,
-    val subscribed: Event.Subscribed,
-    val connectionInfo: MesosClient.ConnectionInfo,
-    /**
-      * Events from Mesos scheduler, sans initial Subscribed event.
-      */
-    val mesosSource: Source[Event, NotUsed])(
-    implicit
-    as: ActorSystem, m: Materializer) extends MesosClient with StrictLoggingFlow {
+class MesosClientImpl(sharedKillSwitch: SharedKillSwitch,
+                      val subscribed: Event.Subscribed,
```

Comment:
Same here. If I'm not wrong we have a simple tab indention.

Reply:
I don't understand... :/

Reply:
agree